src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -22,8 +22,9 @@
 
   <!-- General xunit options -->
   <PropertyGroup>
+    <XunitResultsFileName>testResults.xml</XunitResultsFileName>
     <XunitCommandLine>xunit.console.netcore.exe $(TargetFileName)</XunitCommandLine>
-    <XunitOptions>$(XunitOptions) -xml .\testResults.xml</XunitOptions>
+    <XunitOptions>$(XunitOptions) -xml $(XunitResultsFileName)</XunitOptions>
```

Comment:
Your correct as the tests will be in a different directory so they won't conflict.